plural_rules.go

```diff
@@ -56,7 +57,6 @@ var singleToPlural = map[string]string{
 	"ellipsis":    "ellipses",
 	"equipment":   "equipment",
 	"erratum":     "errata",
-	"faux pas":    "faux pas",
```

Comment:
again, Flect uses parts and this will not work correctly if functions properly support parts. previously, singularize function ignored multi-word strings but now we fixed that.

Like this example, the parts supporting has some weak point that cannot support this kind of thing, but I believe it is more flexible and useful for Buffalo.